src/Dapper.AOT.Analyzers/Internal/Inspection.cs

```diff
@@ -800,7 +919,7 @@ private static bool TryGetAttributeValue<T>(AttributeData? attrib, string name,
                 int index = 0;
                 foreach (var p in ctor.Parameters)
                 {
-                    if (StringComparer.InvariantCultureIgnoreCase.Equals(p.Name == name))
```

Comment:
oof, good catch; that must have been fun to find!

Reply:
you need to thank unit testing for such a treasure ;)